Makefile

```diff
@@ -16,7 +16,7 @@ test:
 test-integ: test
 
 test-docker:
-	curl -s https://raw.githubusercontent.com/sendgrid/sendgrid-oai/HEAD/prism/prism.sh -o prism.sh
+	curl -s https://raw.githubusercontent.com/sendgrid/sendgrid-oai/timeout/prism/prism.sh -o prism.sh
```

Comment:
timeout?

Reply:
trying to fix the tests again...

Reply:
https://github.com/sendgrid/sendgrid-oai/pull/102 should fix them